programs/tensorflowDetection2D/MainDetector.cpp

```diff
@@ -35,8 +35,17 @@
 #include "MainDetector.hpp"
 #include <map>
 
+// Librerias
 
-int maindetector::detect(std::string labels, std::string graph, std::string video_source, yarp::os::Port sender_port_pre, yarp::os::Port sender_port_post) {
+#include <iostream>
+#include <yarp/os/all.h>
+#include <yarp/sig/all.h>
+#include <opencv2/opencv.hpp>
+#include <opencv/cv.h>
+#include <opencv/cvaux.h>
+#include <opencv/highgui.h>
+
+int maindetector::detect(std::string labels, std::string graph, yarp::os::Port sender_port_pre, yarp::os::Port sender_port_post) {
 
 
     std::system("clear");
```

Comment:
Could we remove all the `system` calls?

Reply:
Of course, it´s just to clean terminal.

Reply:
`system` calls removed at ce99c59e4128d1fc6b2ab90ea62901cec0de9c2f.